docs/src/running-tests-python.md

```diff
@@ -49,8 +49,30 @@ You can run a single test, a set of tests or all tests. Tests can be run on one
 
 For more information see [Playwright Pytest usage](./test-runners.md) or the Pytest documentation for [general CLI usage](https://docs.pytest.org/en/stable/usage.html).
 
+## Running Tests
+
+Since Playwright runs in Node.js, you can debug it with your debugger of choice. Playwright comes with the Playwright Inspector which allows you to step through Playwright API calls, see their debug logs and explore [selectors](./selectors.md).
```

Comment:
```suggestion
Since Playwright runs in Python, you can debug it with your debugger of choice with e.g. the [Python extension](https://code.visualstudio.com/docs/python/python-tutorial) in Visual Studio Code. Playwright comes with the Playwright Inspector which allows you to step through Playwright API calls, see their debug logs and explore [selectors](./selectors.md).
```